website/project/model.py

```diff
@@ -2196,6 +2195,8 @@ def fork_node(self, auth, title='Fork of '):
             save=False
         )
 
+        forked.save()
```

Comment:
Yes, without the forked.save()  modularodm raises a `DatabaseError: Record must be loaded`.